### ISSUE TITLE
`run-adapter-static` creates dynamic directories as static folders

### ISSUE THREAD

**Issue by Jack5079** [marko-js/run#64]
The [minimum reproducible example](https://github.com/Jack5079/marko-slug-bug) builds this `dist` folder:

```
dist
├── $
│   └── index.html
├── 404
│   └── index.html
├── path
│   └── index.html
└── index.html
```

But it should be:

```
dist
├── 404
│   └── index.html
├── path
│   └── index.html
└── index.html
```

**Comment by DylanPiercey** [comment]
Good catch @Jack5079, I think we need to add some more checks here https://github.com/marko-js/run/blob/main/packages/adapters/static/src/index.ts#L86

Will try to get to it once I have a sec. If you want to take a stab at a PR that'd be great also 🙏
